idl/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+add_subdirectory(wholeBodyDynamicsSettings)
+add_subdirectory(wholeBodyDynamics_IDLServer)
+add_subdirectory(floatingBaseEstimatorRPC)
```

Comment:
Do you think this `floatingBaseEstimatorRPC` might cause confusion going along with`baseEstimatorV1`?

Reply:
Good point, I need to check.

Reply:
I think there will be no problem, because the `baseEstimatorV1` related RPC service is named as `floatingBaseEstimationRPC`, meanwhile here it is named `floatingBaseEstimatorRPC`. 
I guess this should be harmless.

Reply:
We can eventually remove this if not being used by other users.